### PR TITLE
RN bump

### PR DESCRIPTION
**Pull request by perunt**
Fixes RNBW-4044
Figma link (if any):

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -1,11 +1,14 @@
-platform :ios, '11.0'
```
Comment (terrysahaidak) [inline ios/Podfile:1]: You sure this is required?

File: ios/Podfile [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+platform :ios, '12.4'
 install! 'cocoapods', :deterministic_uuids => false
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
+
 # Prevent Cocoapods from collecting stats, which adds time to each pod installation
 ENV['COCOAPODS_DISABLE_STATS'] = 'true'
 
+production = ENV["PRODUCTION"] == "1"
```
Comment (terrysahaidak) [inline ios/Podfile:10]: Also this change might break things

File: package.json [hunk 3]
```diff
@@ -209,15 +209,15 @@
     "react-native-mail": "4.1.0",
     "react-native-markdown-display": "7.0.0-alpha.2",
     "react-native-minimizer": "brunobar79/react-native-minimizer#26ae6dce10d353de701cebc3a1ae126729bf87f0",
-    "react-native-mmkv": "2.3.3",
+    "react-native-mmkv": "2.4.3",
```
Comment (terrysahaidak) [inline package.json:212]: why this?

File: package.json [hunk 4]
```diff
@@ -209,15 +209,15 @@
...
     "react-native-os": "brunobar79/react-native-os#41a08e45260b8c1c53c77ad00e13ae954cb9337c",
     "react-native-pager-view": "5.4.24",
     "react-native-palette-full": "1.2.0",
     "react-native-permissions": "3.0.5",
     "react-native-quick-md5": "3.0.3",
     "react-native-radial-gradient": "rainbow-me/react-native-radial-gradient#1266cd8f6ac3321bcf27357f8771bd088f9f27ef",
     "react-native-randombytes": "3.5.3",
-    "react-native-reanimated": "2.8.0",
+    "react-native-reanimated": "2.9.1",
```
Comment (terrysahaidak) [inline package.json:220]: why this?

File: src/components/coin-icon/RequestCoinIcon.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React, { useCallback } from 'react';
 import { CircularProgress } from 'react-native-circular-progress';
 import { useTheme } from '../../theme/ThemeContext';
+
```
Comment (terrysahaidak) [inline src/components/coin-icon/RequestCoinIcon.js:4]: why this

**Review by terrysahaidak — COMMENTED** [review]
Few comments

File: package.json (hunk 3 above)
Comment (perunt) [reply package.json:212]: I need [this](https://github.com/mrousavy/react-native-mmkv/commit/8af0d47c2e09e52e95e5ab240971ca0821568b0a) changes
I didn't fount wrong behavior but if it critical, i can make a patch

File: package.json (hunk 4 above)
Comment (perunt) [reply package.json:220]: the same here, support for RN 69+ added from [2.9.0](https://github.com/software-mansion/react-native-reanimated/releases/tag/2.9.0)

File: src/components/coin-icon/RequestCoinIcon.js (hunk 5 above)
Comment (perunt) [reply src/components/coin-icon/RequestCoinIcon.js:4]: accidentally

File: package.json (hunk 4 above)
Comment (terrysahaidak) [reply package.json:220]: I guess it should be [2.9.1](https://github.com/software-mansion/react-native-reanimated/releases/tag/2.9.1) now

File: ios/Podfile (hunk 2 above)
Comment (perunt) [reply ios/Podfile]: removed

**Review by estrattonbailey — COMMENTED** [review]
@perunt what would it take to get this PR merged, or should we close, prioritize, and create a new PR later?

Also, are all these patches auto-generated? Just curious.

**Comment by terrysahaidak** [comment]
@estrattonbailey this PR had been postponed in favor of the Android GA project, since we didn't want to introduce any other bugs related to all the library updates.

This PR probably can be closed since we'll be upgrading to rn 70 now anyway.

**Comment by estrattonbailey** [comment]
Works for me! Let's close it then. We can always re-open later 👍

**Pull request closed without merging**